src/main/java/com/squareup/javapoet/JavaFile.java

```diff
@@ -237,6 +268,7 @@ public Builder indent(String indent) {
     }
 
     public JavaFile build() {
+      Collections.sort(staticImports);
```

Comment:
I think it’s slightly neater to keep it in a sorted set, but it doesn’t really matter!